gtfs2netexfr/README.md

```diff
@@ -1,10 +1,10 @@
 # `gtfs2netexfr`
 
-Command-Line Interface to convert [GTFS] data format into [NeTEx] France data
+Command-Line Interface to convert [GTFS] data format into [NeTEx]-France data
 format.
 
-[GTFS]: https://developers.google.com/transit/gtfs/reference/
-[NeTEx]: http://netex-cen.eu/
+[GTFS]: https://gtfs.org/reference/static
```

Comment:
I'm not sure we should change that. What do you think @prhod ?

Reply:
As discussed, gtfs.org is now the reference, handled by MobilityData and not only Google.